lib/pages/new_group/new_group.dart

```diff
@@ -54,6 +60,7 @@ class NewGroupController extends State<NewGroup> with ComparablePresentationCont
     searchContactsController.dispose();
     fetchContactsController.dispose();
     groupNameTextEditingController.dispose();
+    createRoomStreamController.dispose();
```

Comment:
yeah cool, What do you think about the problem when closing this screen before the API complete?

Reply:
This controller needs to be closed later so I decided to close it after the action finishes and not on widget dispose. 
Create group without avatar : Move to new room + cancel stream
Create group with avatar : Move to new room + change avatar => if avatar succeed : cancel stream if not throw error but room still created